docs/user_guide/plugin_development/backend_specifics.rst

```diff
@@ -10,10 +10,11 @@ Additionally, interacting directly with the bot internals gives you the freedom
 to control Errbot in highly specific ways that may not be officially supported.
 
 .. warning::
-	The following instructions describe how to interface directly with the underlying bot object and clients of backends.
```

Comment:
My IDE noticed these as inconsistent use of tabs vs spaces in the rest of the file

Reply:
no prob